### ISSUE TITLE
LFV/SFV, SHV/LHV RSP Vector Load/Store ops

### ISSUE THREAD

**Issue by hcs64** [n64dev/cen64#149]
I wasn't sure if these were not actually on release hardware, but they seem to work as expected on my N64. I'll take a crack at implementing these soon if it isn't planned.

**Comment by Mastergatto** [comment]
Yes, they are implemented on the real hardware, in fact these opcodes are used by Resident Evil 2: https://github.com/n64dev/cen64/issues/64

I'm not a developer, but I'd say to go ahead anyways, as CEN64 is seeing really low activity these days.
